### ISSUE TITLE
[Search Map] make filter sets available on searchmap

### ISSUE THREAD

**Issue by capoaira** [2Abendsegler/GClh#1504]
The new map includes the same filter like the new search. It would nice to use own filtersets from the new search also on the new map.
![grafik](https://user-images.githubusercontent.com/50927164/99937924-127d1200-2d67-11eb-9bc9-06ca355520ac.png)

**Comment by 2Abendsegler** [comment]
@capoaira 
Das Issue würde ich auch gerne schließen. Soweit ich mich erinnere war die Einschätzung, dass der Aufwand ziemlich groß ist, weil so ziemlich alles neu entwickelt werden muss um es auf der Search Map und auf der Search Seite gleichzeitig zu verwenden. Zudem war das Feature ziemlich anfällig und wir mussten alle paar Monate daran basteln. Das mag aber mit der Art und Weise zu tun haben wie es implementiert war.
  
Ich würde das Issue also gerne schließen und es mit dem Label "storage" versehen. Was meinst du? 

(Mit dem Label "storage" kennzeichnen wir geschlossene Issues die wir aus unterschiedlichen Gründen im Moment nicht umsetzen, die es aber verdient haben umgesetzt zu werden.)

**Comment by capoaira** [comment]
Ja, können wir erstmal schießen, wenn wir #2055 fixen, können wir dieses Issue vermutlich gleich und ohne viel Mehraufwand mit einbauen.
